root/static/scripts/edit/URLCleanup.js

```diff
@@ -3180,6 +3180,41 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'librivox': {
+    match: [new RegExp('^(https?://)?(www\\.)?librivox\\.org', 'i')],
+    restrict: [LINK_TYPES.downloadfree],
+    clean: function (url) {
+      return url.replace(/^(?:https?:\/\/)?(?:www\.)?librivox\.org\/((?:author|reader)\/[\d]+|[\w\d-]+)(?:[/?#].*)?$/, 'https://librivox.org/$1');
```

Comment:
Minor untested simplification:

```suggestion
      return url.replace(/^(?:https?:\/\/)?(?:www\.)?librivox\.org\/((?:author|reader)\/[\d]+|[\w\d-]+).*$/, 'https://librivox.org/$1');
```